src/taskie/Taskie.java

```diff
@@ -17,7 +17,7 @@ public static void main(String[] args) {
 	}
 
 	public Taskie(String[] args) {
-		Controller = new taskie.controller.Controller();
+		Controller = new taskie.controller.Controller(UI);
```

Comment:
Use Taskie.UI in Controller instead of passing the variable in to maintain consistency.

Please see https://github.com/cs2103jan2015-t15-2j/main/blame/master/src/taskie/parser/CommandParser.java#L120